app/views/incidents/upload.html.erb

```diff
@@ -24,13 +24,19 @@
     <%= form_tag({action: :upload}, id: 'upload-control', multipart: true) do %>
       <label class="btn btn-file btn-bayes">
         <span id="fileButtonText">Choose file to upload <i class="fa fa-arrow-circle-up"></i></span>
-        <input type="file" name="file" style="display: none;" accept="application/json">
+        <input type="file" name="file" style="display: none;" accept="application/json,application/xml">
       </label>
     <% end %>
 
     <div id="schema" class="block">
```

Comment:
I have no real preference. I do kind of like the output block being on top because we do have a convention of alerts/notices going on top of the page, and the output is sort of an alert.